app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :username %><br />
+    <%= f.text_field :username, autofocus: true, autocomplete: "email" %>
+  </div>
+  
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Please consider testing the registration form again, when I register a user I get this error on the browser but the user is added to the database.

- [ ] Also in the database, the `name` field is not added to the database when a user is registered, please check on this since as per the ERD diagram then `Name` field is a mandatory field

## Error on sign up
![Screenshot from 2022-10-04 22-31-38](https://user-images.githubusercontent.com/16392046/193911376-6c0b2815-ec3e-4c02-b162-869a4de743b4.png)

## Name field in the database is missing


![Screenshot from 2022-10-04 22-47-41](https://user-images.githubusercontent.com/16392046/193912048-d1a25549-0a7f-4f0c-9a0f-270082d608c6.png)